website/docs/foundations/focus-ring/partials/code/showcase.md

```diff
@@ -0,0 +1,30 @@
+<!--
+<section data-test-percy data-section="showcase">  
+  <h4 class="dummy-h4">Focus ring:</h4>
+  <p class="dummy-paragraph">Standalone "focus ring" effect</p>
+  <div class="dummy-focus-ring-samples">
+    <div class="dummy-focus-ring-sample hds-focus-ring-action-box-shadow">
+      <Doc::Placeholder @text="no radius" @width="100" @height="100" @background="transparent" />
+    </div>
+    <div class="dummy-focus-ring-sample dummy-focus-ring-sample--border-radius hds-focus-ring-action-box-shadow">
+      <Doc::Placeholder @text="with border radius" @width="100" @height="100" @background="transparent" />
+    </div>
+  </div>
+  <div class="dummy-focus-ring-samples">
+    <div class="dummy-focus-ring-sample">
+      <span class="dummy-text-small">action</span>
+      <br />
+      <div class="hds-focus-ring-action-box-shadow">
+        <Doc::Placeholder @text="with border radius" @width="100" @height="100" @background="transparent" />
+      </div>
+    </div>
+    <div class="dummy-focus-ring-sample">
+      <span class="dummy-text-small">critical</span>
+      <br />
+      <div class="hds-focus-ring-critical-box-shadow">
+        <Doc::Placeholder @text="with border radius" @width="100" @height="100" @background="transparent" />
+      </div>
+    </div>
+  </div>
+</section>
+-->
```

Comment:
Is there a reason we want to comment this chunk out vs. just deleting it?

Reply:
We don't want to delete it because we may want it in the future when we decide what to do with the showcase, but it doesn't need to be commented out since the reference to the showcase.md file is already commented out.